Sources/Misc/Deprecations.swift

```diff
@@ -118,7 +118,7 @@ public extension Purchases {
             userDefaults: userDefaults,
             platformInfo: nil,
             responseVerificationMode: .default,
-            storeKit2Setting: .init(useStoreKit2IfAvailable: useStoreKit2IfAvailable),
+            storeKitVersion: useStoreKit2IfAvailable ? .storeKit2 : .default,
```

Comment:
I know this is deprecated, but should this be
```suggestion
            storeKitVersion: useStoreKit2IfAvailable ? .storeKit2 : .storeKit1,
```

If you pass `false`, it might still use it SK2 if/when that becomes the default.